### PR TITLE
initial work for curator chart

### PR DESCRIPTION
**Pull request by DStorck**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,15 @@
...
+* Iterating through a list of user-defined [filters](https://www.elastic.co/guide/en/elasticsearch/client/curator/current/filters.html) to progressively remove indices from the *actionable* list as needed.
 
-Using template-chart
-====================
+Our Chart deletes all logstash-prefixed indices older than 14 days.
 
-First, clone this repo to be named chart-${new_repo_name} and check it out
-(eg chart-sample-chart).
+## Installation
+`$ helm repo add cnct http://atlas.cnct.io`
```
Comment (davidewatson) [inline README.md]: Is this how quay.io apps are installed? If so, the template chart convention says this should be `https://quay.io/repository/samsung_cnct/chart-curator` for the application on quay, and `https://quay.io/repository/samsung_cnct/curator` for the container.

Do we like this? Would we prefer `curator-container` and `curator` instead? Containers and apps in quay share the same namespace, so we must have two names...

File: curator-chart/.helmignore [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Patterns to ignore when building packages.
```
Comment (davidewatson) [inline curator-chart/.helmignore]: Nice! This probably should be pushed up to the `template-chart` repo.

File: curator-chart/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# A Helm Chart for Curator
+
+## Installation
+
+Install this in your Kubernetes cluster with [Helm](https://github.com/kubernetes/helm):
+
+```
+helm repo add cnct http://atlas.cnct.io
```
Comment (davidewatson) [inline curator-chart/README.md]: Same comment from earlier.

File: curator-chart/.helmignore (hunk 2 above)
Comment (DStorck) [reply curator-chart/.helmignore]: @davidewatson happy to do that.  Does it also make sense to include other scaffolding, like the templates folder, _helpers.tpl file, or values.yaml?

Comment (DStorck) [reply curator-chart/.helmignore]: okay, that has all been done.

File: curator-chart/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+Add the following to your [Kraken](https://github.com/samsung-cnct/kraken-lib) configuration template:
+```
+helmConfigs:
+  - &defaultHelm
+    name: defaultHelm
+    kind: helm
+    repos:
+      -
+        name: atlas
+        url: http://atlas.cnct.io
```
Comment (davidewatson) [inline curator-chart/README.md]: https://quay.io

Comment (DStorck) [reply curator-chart/README.md]: i changed it below for the charts: section, but left it alone as in the repo part, just to model what the config could look like.

**Pull request merged**